doc/source/tutorial/stats/continuous_studentized_range.rst

```diff
@@ -0,0 +1,27 @@
+
+.. _continuous-studentized_range:
+
+Studentized Range Distribution
+==============================
+This distribution has two shape parameters :math:`\nu>0` and :math:`k>0` and support is :math:`q\geq0`.
```

Comment:
Other distributions (Student's T is a good example) do the same thing, with `v` representing DOF in doc, and some other (usually df) symbol representing it in code.

Reply:
Other functions use _v_ in their doc to represent DOF and `df` within code, so I did the same. For tests, I think it makes sense to use `v` rather than `df` because we're testing inputs, whose names (I think) should match what's in the doc as closely as possible.